<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>                                          MSRIT Branch Reallotment</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cGPALabel">
        <property name="text">
         <string>CGPA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cGPALineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="currentBrachLabel">
        <property name="text">
         <string>Current Branch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="currentBrachComboBox">
        <item>
         <property name="text">
          <string>Biotechnology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chemical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electrical &amp; Electronics Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronics &amp; Communication Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Industrial Engineering &amp; Management</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Information Science &amp; Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mechanical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medical Electronics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Telecommunication Engineering</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="firstPreferenceLabel">
        <property name="text">
         <string>First Preference</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="currentBrachComboBox_2">
        <item>
         <property name="text">
          <string>Biotechnology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chemical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electrical &amp; Electronics Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronics &amp; Communication Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Industrial Engineering &amp; Management</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Information Science &amp; Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mechanical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medical Electronics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Telecommunication Engineering</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="secondPreferenceLabel">
        <property name="text">
         <string>Second Preference</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="currentBrachComboBox_3">
        <item>
         <property name="text">
          <string>Biotechnology</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chemical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer Science and Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electrical &amp; Electronics Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Electronics &amp; Communication Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Industrial Engineering &amp; Management</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Information Science &amp; Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mechanical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medical Electronics</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Telecommunication Engineering</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="done">
      <property name="text">
       <string>Allocate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>Next</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
